tools/tests/com.liferay.ide.server.core.tests/META-INF/MANIFEST.MF

```diff
@@ -12,7 +12,8 @@ Require-Bundle: org.eclipse.core.runtime,
  com.liferay.ide.server.tomcat.core;bundle-version="2.1.0",
  org.apache.commons.io;bundle-version="2.0.1",
  org.eclipse.jst.server.tomcat.core,
- org.eclipse.m2e.maven.runtime
+ org.eclipse.m2e.maven.runtime,
+ org.eclipse.ui.console
```

Comment:
ui.console can't be added to server.core tests.  Only server.ui.tests or project.ui.tests.